src/main/java/com/mineboundteam/minebound/capabilities/PlayerUtilityToggleProvider.java

```diff
@@ -53,17 +53,21 @@ public static void UpdatePlayerSync(Player oldPlayer, Player newPlayer){
     }
     public static class UtilityToggle {
         public boolean fire = false;
+        public boolean earth = false;
```

Comment:
should this one default to on?

Reply:
I actually don't really understand the functionality of Earth Utility from the SRS. Is it like Earth Defensive but with a set mining speed + vein miner?

Reply:
> should this one default to on?

Yeah Probably I'd rather have vein miner on initially

Reply:
> 

I think just vein miner vs. regular mining

Reply:
ooh wait does it include vein miner? alright, in that case nevermind. 

It'll add mining speed and vein miner when breaking a block. Vein miner if the toggle is on, mining speed regardless

Reply:
Got it, ok. We don't mention the increase to the mining speed but I'll add that to the SRS.